qelem/lib/screens/profile/EditProfile.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class EditProfile extends StatefulWidget {
+  const EditProfile({Key? key}) : super(key: key);
+
+  @override
+  State<EditProfile> createState() => _EditProfileState();
+}
+
+class _EditProfileState extends State<EditProfile> {
+  TextEditingController userNameController = TextEditingController();
+  TextEditingController firstNameController = TextEditingController();
+  TextEditingController lastNameController = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    TextStyle? textStyle = Theme.of(context).textTheme.subtitle1;
+    return Scaffold(
+        body: Column(children: [
+      Container(
+          margin: const EdgeInsets.only(top: 25.0),
+          child: Align(
+            alignment: Alignment.topCenter,
+            child: Stack(
+              alignment: const Alignment(1.1, 1.1),
+              children: [
+                const CircleAvatar(
+                  backgroundImage: AssetImage("abebe.jpeg"),
+                  radius: 80,
```

Comment:
You can remove the radius, it is redundant.

Reply:
It's not a border radius, it's the actual radius of the circle. It determines the size.